src/doc/dev_manual/LiveIncidentResponse.rst

```diff
@@ -0,0 +1,121 @@
+    across the development team.
+  * To achieve a 1-hour response time.
+  * To reduce incident response interruptions for the team as a whole.
+
+For the VisIt_ project, normal business hours are 8am-12pm and 1-5pm, Monday
+through Friday excluding LLNL holidays. The work of *live* customer support is
+allocated in one-week intervals. During a week, one developer is assigned
+as the *primary* customer support contact and a second developer is assigned
+as a *backup*. Live customer support issues will be logged and tracked in a
+separate GitHub respository within the
+`visit-dav GitHub organization. <https://github.com/visit-dav>`_
+
+
+The Primary's Role
+------------------
+
+The primary's role is to answer all incidents that occur during that week
+including those that came in during the preceding weekend/holiday. Other
+developers who may be contacted directly by customers should redirect such
+inquires to the primary. The primary's goal is to *wrap-up* all incidents
+by the end of their week. If an issue cannot be wrapped-up, it gets handed-off
+to next week's primary. Handoff's of incidents are managed formally with
+an email to customer and the next week's primary and backup contact information.
+
+For each new incident, the primary should file an issue ticket in the
+`visit-live-incidents <https://github.com/visit-dav/visit-live-incidents/issues>`_
+repo and assign themselves. When the incident is resolved, the associated issue
+should be closed. Primary should endeavour to capture all relevant information
+and communications involving the incident in this issue. For users who do not have
+GitHub accounts, this will require the primary to do a modest amount of cutting
+and pasting between email and GitHub issues.
+
+The goal of the primary is to reach a *resolution* of each incident. Resolution
+often involves one or more of the following activities.
+
+  * Answering a question or refering user to documentation.
+  * Determining if user's issue is known (e.g. an issue ticket already exists).
+  * Updating a known issue with additional information from this user, perhaps
+    reprioritizing the issue or putting the ticket back into the UN-reviewed
+    state for further discussion at a VisIt_ project meeting.
+  * Identifying and subsequently fixing a *low-hanging fruit* issue.
+  * Developing a reproducer for developers.
+  * Developing a work-around for users.
+  * Identifying and filing a coding *new* issue in the main source repo.
+
+Another role of the primary is to use their time during the week to fix any
+*low-hanging fruit* issues revealed by an incident. For example, if an incident
+reveals incorrect or missing documentation, the primary should work to fix such
+issues. As a rule of thumb, low-hanging fruit is considered to be anything that
+primary believes is fixable within a half-day's (4 hours) worth or effort. 
+
+When primary has challenges with incident diagnosis, s/he should feel free but
```

Comment:
The extra phrase "but should otherwise sparingly" makes this sentence a little hard to understand. I think you're trying to emphasize that involving extra developers should only happen under certain conditions, which is important to note, but maybe there is a better way to say that.